plugins/wfp/common.py

```diff
@@ -275,14 +275,17 @@ def map_assistance_step(self, step, given_assistance):
             given_medication = self.split_given_medication(step.get("medication_2"), quantity)
             given_assistance = given_assistance + given_medication
 
-        if step.get("ration_to_distribute") is not None:
+        if step.get("ration_to_distribute") is not None or step.get("ration") is not None:
             quantity = 0
             if step.get("_total_number_of_sachets") is not None:
                 quantity = step.get("_total_number_of_sachets", 0)
             elif step.get("_csb_packets") is not None:
                 quantity = step.get("_csb_packets", 0)
 
-            assistance = {"type": step.get("ration_to_distribute"), "quantity": quantity}
```

Comment:
@butofleury check if it's correct to pass quantity on ration_to distribute.

Reply:
@micnice , for some forms, it's **ration_to_distribute** . That's why I've added it to make sure to take into account all cases!